### ISSUE TITLE
map and naming the resulting column

### ISSUE THREAD

**Issue by tcovert** [JuliaComputing/IndexedTables.jl#52]
Is there a way to assign a name to the resulting column that map creates?  I would have thought `@NT()` would do this but it seems not to exactly work:

```
julia> tbl = IndexedTable(Columns(idx = collect(1:10)), Columns(vals = randn(10)))
julia> map(x->abs.(x.vals), tbl)
julia> map(x->@NT(aval = abs.(x.vals)), tbl)
```
the first map statement gives the output I'd expect but no name for the data column.  the second map statement does have a name, but it gives me an array of named tuples as opposed to an array of values.

thanks in advance for any suggestions.

**Comment by shashi** [comment]
I agree that when the function returns a tuple or named tuple, `map` should construct new `Columns`...  I've made a PR for this #54

**Comment by shashi** [comment]
Closed by #57
